### PR TITLE
add polythene@0.2.31 w/ npm auto-update

### LINKED ISSUE #8437: [Request] Add polythene
**Library name:** Polythene 
**Git repository url:** https://github.com/ArthurClemens/Polythene
**License(s):** MIT 
**Official homepage:** http://polythene.js.org/

### PR DESCRIPTION
**Pull request by nolsherry**
@pvnr0082t, please help me review this PR, thanks!

Checklist for **Pull request** or **lib adding request issue** follows the conventions.

Note that if you are using a distribution purpose repository/package, please also provide the url and other related info like popularity of the source code repo/package.

# Profile of the lib
 * Git repository (required): https://github.com/ArthurClemens/Polythene
 * Official website (optional, not the repository): https://polythene.js.org/
 * NPM package url (optional): https://www.npmjs.com/package/polythene
 * GitHub / Bitbucket popularity (required):
   - Count of stars: 155
   - Count of watchers: 17
   - Count of forks: 12
 * NPM download stats (optional):
   - Downloads in the last day: 5
   - Downloads in the last week: 16
   - Downloads in the last month: 382

# Essential checklist
 * [ ] I'm the author of this library
   * [ ] I would like to add link to the page of this library on CDNJS on website / readme
 * [X] This lib was not found on cdnjs repo
 * [X] No already exist / duplicated issue and PR
 * [X] The lib has notable popularity
   * [X] More than 100 [Stars / Watchers / Forks] on [GitHub / Bitbucket]
   * [ ] More than 500 downloads stats per month on npm registry
 * [X] Project has public repository on famous online hosting platform (or been hosted on npm)

# Auto-update checklist
 * [X] Has valid tags for each versions (for git auto-update)
 * [X] Auto-update setup
 * [X] Auto-update target/source is valid.
 * [X] Auto-update filemap is correct.

# Git commit checklist
 * [x] The first line of commit message is less then 50 chars, be clean and clear, easy to understand.
 * [x] The parent of the commit(s) in the PR is not old than 3 days.
 * [x] Pull request is sending from a non-master branch with meaningful name.
 * [x] Separate unrelated changes into different commits.
 * [x] Use rebase to squash/fixup dummy/unnecessary commits into only one commit.
 * [x] Close corresponding issue in commit message
 * [x] Mention related issue(s), people in commit message, comment.

close #8437, cc @ArthurClemens

### CODE REVIEW

**Comment by PeterDaveHello** [comment]
Do we need `polythene.es6`?

File: ajax/libs/polythene/package.json [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+  "dependencies": {
+    "fastclick": "^1.0.6",
+    "j2c": "^0.11.2"
+  },
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/ArthurClemens/Polythene.git"
+  },
+  "author": "Arthur Clemens",
+  "licenses": [
```
Comment (pvnr0082t) [inline ajax/libs/polythene/package.json]: @nolsherry Why did you add licenses field? There is license info at line 6.

**Comment by pvnr0082t** [comment]
@nolsherry Please add https://polythene.js.org/ as homepage field

**Comment by ArthurClemens** [comment]
Polythene has a modular structure, roughly comparable to lodash. Javascript modules can include parts on "as need" base.

To include all of Polythene at once, use `polythene-standalone.js`.

**Comment by PeterDaveHello** [comment]
Thanks @ArthurClemens

**Comment by nolsherry** [comment]
@pvnr0082t, I have added `https://polythene.js.org` as homepage field, please help me review the PR, thanks!

**Comment by pvnr0082t** [comment]
@nolsherry Please pass the tests first.

**Comment by nolsherry** [comment]
@pvnr0082t, I have passed the tests, please help me review this PR, thanks!

**Comment by pvnr0082t** [comment]
@nolsherry You don't fix the licenses problem.

**Comment by nolsherry** [comment]
@pvnr0082t, I have removed the duplicate `license` field, please help me review this PR, thanks!

**Comment by pvnr0082t** [comment]
@PeterDaveHello I think this PR is ok. Please review it again, thank you.

**Comment by PeterDaveHello** [comment]
@nolsherry you should do one more rebasing with the latest master branch before you sent the updates next time.
